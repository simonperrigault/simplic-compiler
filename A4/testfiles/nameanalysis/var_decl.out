Error at line 3: symbol 'a' is already declared!
