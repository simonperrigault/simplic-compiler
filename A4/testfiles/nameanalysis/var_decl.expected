Error at line 3:9 : symbol 'a' is already declared!
